### PR TITLE
Add a CLI tool to extend the RTMRs.

### PR DESCRIPTION
**Pull request by qinkunbao**

### CODE REVIEW

**Review by jrjatin — APPROVED** [review]

File: tools/extend/README.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+
+### `verbosity`
+
+Used to set the verbosity of logger, where higher number means more verbose output.
+
+Default value is `0`.
+
+## Examples
+
+The following example measures an eventlog and extends it measurement into the RTMR2
```
Comment (ivishwesh) [inline tools/extend/README.md:32]: Minor changes:
eventlog --> event log , for consistency with line 18
it --> its

File: tools/extend/README.md [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+
+The tool's output is an error or "Success".
+
+## Usage
+
+```
+./extend [options...]
+```
+
+### `-in`
```
Comment (ivishwesh) [inline tools/extend/README.md:16]: You have used a - here but not for the other flags. Please check.

File: tools/extend/README.md [hunk 3]
```diff
@@ -0,0 +1,42 @@
+# `check` CLI tool
+
+This binary is a thin wrapper around the `rtmr` library to
+extend the measurement into RTMR registers.
+
+The tool's input is the event log.
+
+The tool's output is an error or "Success".
+
+## Usage
```
Comment (ivishwesh) [inline tools/extend/README.md:10]: The rtmr flag is used in the example but not listed / described over here.

File: tools/extend/README.md (hunk 2 above)
Comment (qinkunbao) [reply tools/extend/README.md:16]: Oh, I followed the readme file from other documents such as https://github.com/google/go-tdx-guest/blob/main/tools/attest/README.md. Looks like we need to change them as well. Thanks for the finding.

File: tools/extend/README.md (hunk 3 above)
Comment (qinkunbao) [reply tools/extend/README.md:10]: Added

File: tools/extend/README.md (hunk 2 above)
Comment (qinkunbao) [reply tools/extend/README.md:16]: https://github.com/google/go-tdx-guest/pull/45 is created

File: tools/extend/README.md [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+
+This flag provides the path to the event log to measure. Stdin is "-".
+
+### `-rtmr`
+
+This flag defines the RTMR index that will be extended. As specified in the Intel TDX specification, user space applications can only extend to RTMR2 or RTMR3.
+
+### `-quiet`
+
+If set, doesn't write exit errors to Stdout. All results are communicated through exit code.
```
Comment (kongoshuu) [inline tools/extend/README.md:26]: Maybe "If set, no errors are written to stdout. Results are communicated through exit code only"

**Review by ivishwesh — APPROVED** [review]

File: tools/extend/extend.go [hunk 5]
```diff
@@ -0,0 +1,87 @@
...
+)
+
+const (
+	// Exit code 1 - tool usage error.
+	exitTool = 1
+)
+
+var (
+	infile    = flag.String("in", "-", "Path to the input event log. Stdin is \"-\".")
+	quiet     = flag.Bool("quiet", false, "If true, writes nothing the stdout or stderr. Success is exit code 0, failure exit code 1.")
```
Comment (kongoshuu) [inline tools/extend/extend.go:36]: typo? Do you mean "write nothing to stdout or stderr"?

**Review by kongoshuu — APPROVED** [review]

**Pull request merged**